android-components/components/concept/storage/src/main/java/mozilla/components/concept/storage/CreditCardsAddressesStorage.kt

```diff
@@ -353,9 +351,7 @@ data class UpdatableCreditCardFields(
 @Parcelize
 data class Address(
     val guid: String,
-    val givenName: String,
-    val additionalName: String,
-    val familyName: String,
+    val name: String,
```

Comment:
Could we update this from `name` to `fullName` or `completeName`? At a minimum I think we should update the description to something such as "A person's full name, typically made up of a first name, any middle names and surname, e.g. John Joe Doe"